docs/notebooks/pmp.ipynb

```diff
@@ -0,0 +1,493 @@
+{
```

Comment:
<p>This approach is fine for now, but there will be major changes to the testing data fetching mechanisms in the next few versions, using <code>pooch</code> with a registry. This will mean that pointing to a commit will not be needed.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/hydrologie/xhydro/pull/176/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
This will be addressed after the merge of #212